### PR TITLE
Minor corrections to v4 GQL Armor and Auth docs

### PR DESCRIPTION
**Pull request by talk2MeGooseman**
Reading through the new docs found a couple of readability corrections that could be made. Solid docs, as to be expected!

### CODE REVIEW

File: docs/docs/graphql.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ There's three types of GraphQL operations, but here we'll only focus on two: Que
 The following GraphQL query:
 
 ```graphql
-query GetProject {
+query GetProjectl
```
Comment (jtoar) [inline docs/docs/graphql.md:20]: Was this one a mistake? The original seems ok

File: docs/docs/graphql.md [hunk 2]
```diff
@@ -1504,9 +1504,9 @@ Limit the number of aliases in a document. Defaults to 15.
 
 ##### Example
 
-Aliases allow you rename the data that is returned in a query’s results. They manipulate the structure of the query result that is fetched from your service, displaying it according to your web component's needs.
+Aliases allows you to rename the data that is returned in a query’s results. They manipulate the structure of the query result that is fetched from your service, displaying it according to your web component's needs.
```
Comment (jtoar) [inline docs/docs/graphql.md:1507]: I think "allow" was correct but definitely need the "to":

```suggestion
Aliases allow you to rename the data that is returned in a query’s results. They manipulate the structure of the query result that is fetched from your service, displaying it according to your web component's needs.
```

**Review by jtoar — CHANGES_REQUESTED** [review]
Thanks @talk2MeGooseman, left a few comments

File: docs/docs/graphql.md (hunk 1 above)
Comment (talk2MeGooseman) [reply docs/docs/graphql.md:20]: Thanks, I tried to do these edits via the web interface. DO NOT RECOMMEND

Comment (talk2MeGooseman) [reply docs/docs/graphql.md]: Fixed

**Review by jtoar — APPROVED** [review]

File: docs/docs/graphql.md (hunk 2 above)
Comment (talk2MeGooseman) [reply docs/docs/graphql.md]: Fixed

**Pull request merged**